### PR TITLE
Added demonic gorilla AoE warnings

### PR DESCRIPTION
**Pull request by Seraph2**

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java [hunk 1]
```diff
@@ -330,6 +330,8 @@ private boolean isConfigEnabledForProjectileId(int projectileId, boolean notify)
 				return notify ? config.isDrakeNotifyEnabled() : config.isDrakeEnabled();
 			case CERB_FIRE:
 				return notify ? config.isCerbFireNotifyEnabled() : config.isCerbFireEnabled();
+			case DEMONIC_GORILLA_BOULDER:
+				return config.isDemonicGorillaEnabled();
```
Comment (Infinitay) [inline runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: Coming from the initial comment, here you'd want to follow the same guidelines and return the _same_ value

ie `notify ? config.isDemonicGorillaNotifyEnabled() : config.isDemonicGorillaEnabled();`

Comment (ThatGamerBlue) [reply runelite-client/src/main/java/net/runelite/client/plugins/aoewarnings/AoeWarningPlugin.java]: > Coming from the initial comment, here you'd want to follow the same guidelines and return the _same_ value
> 
> ie `notify ? config.isDemonicGorillaNotifyEnabled() : config.isDemonicGorillaEnabled();`

**Comment by Seraph2** [comment]
I have added the a notify to demonic gorilla now.

**Comment by Kyleeld** [comment]
made a quick change to trigger travis rebuild. travis is now fixed and PR should complete successfully.

**Pull request merged**